src/search/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import SearchBar from './headerBar';
+import Result from './result';
+
+const navigationOptions = {
+  header: () => <SearchBar />,
+};
+
+const search = () => (
+  <Result />
+);
+
+search.navigationOptions = navigationOptions;
+
+export default search;
```

Comment:
It is the same as you would export the Result the component directly after setting its `navigationOptions` property